corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -43,6 +43,63 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
         $profilesTab.koApplyBindings(profileManager);
     }
 
+    var appReleaseLogsModel = function () {
+        let self = {};
+        self.releaseLogs = ko.observableArray();
+        self.fetchLimit = ko.observable();
+        self.totalItems = ko.observable();
+        self.fetchState = ko.observable();
+
+        self.onPaginationLoad = function () {
+            self.goToPage(1);
+        };
+
+        self.goToPage = function (page) {
+            if (self.fetchState() === 'pending') {
+                return false;
+            }
+            self.fetchState('pending');
+            $.ajax({
+                url: self.reverse("paginate_release_logs"),
+                dataType: 'json',
+                data: {
+                    page: page,
+                    limit: self.fetchLimit,
+                },
+                success: function (data) {
+                    self.releaseLogs(
+                        _.map(data.app_release_logs, function (log) {
+                            log.status = log.is_released ? "Released" : "Testing";
+                            log.timestamp = new Date(log.timestamp).toLocaleString();
+                            return ko.mapping.fromJS(log);
+                        })
+                    );
+                    self.totalItems(data.pagination.total);
+                    self.fetchState('');
+                },
+                error: function () {
+                    self.fetchState('error');
+                },
+            });
+        };
+        self.reverse = function () {
```

Comment:
What is this function for? A better name would be useful to highlight its purpose

Reply:
This function is used for getting the endpoint associated with a slug. Renamed it to getApiEndpoint.

Reply:
Why does this function exist? Can't you call `hqImport("hqwebapp/js/initial_page_data").reverse("paginate_release_logs")` where you're currently calling `self.getApiEndpoint("paginate_release_logs")`?